package.json

```diff
@@ -41,7 +41,7 @@
     "enzyme-to-json": "^3.2.2",
     "flow-bin": "^0.60.1",
     "husky": "^0.14.3",
-    "jest": "^21.2.1",
+    "jest": "^22.1.4",
```

Comment:
for a moment I thought there was no test setup in this package (confused it with unstated repo), so I've started creating one and I've bumped jest version by doing so - can revert that change if you dont like it